iroha_python/test.py

```diff
@@ -12,6 +12,6 @@
 cl = Client(cfg)
 
 domain = Domain("xor")
-register = Instruction(RegisterBox(Expression(Value(IdentifiableBox(domain)))))
+register = Instruction(Register(Expression(Value(Identifiable(domain)))))
 
 cl.submit_inst(register)
```

Comment:
Maybe it is better to also create a test for events?

Reply:
Check out `ws.py` file

Reply:
But it does not seem like a test, nothing is checked there. We should add some assertions.